language.md

```diff
@@ -126,5 +126,6 @@ Se vi parolas esperanton, alklaki [tie] (esperanto/komenci.md).
 ئەگەر ئۇيغۇرچە سۆزلىسىڭىز، [بۇ يەرگە كىرىڭ](uyghurish/uyghurish-story.md)
 
 Na kama lugha yako ni Swahili, [click hapa](swahili/swahili.md)
-=======
-Kama unazungumza kiswahili [ingia hapa](swahili/marshmallow.md)
+=======
+Kama unazungumza kiswahili [ingia hapa](swahili/marshmallow.md)
+If you speak cricket go here [click](hindi/cricket/cricket.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.